server/src/main/java/org/opensearch/cluster/coordination/Coordinator.java

```diff
@@ -821,7 +825,15 @@ boolean publicationInProgress() {
     protected void doStart() {
         synchronized (mutex) {
             CoordinationState.PersistedState persistedState = persistedStateSupplier.get();
-            coordinationState.set(new CoordinationState(getLocalNode(), persistedState, electionStrategy));
+            coordinationState.set(
+                new CoordinationState(
```

Comment:
can we simply PersistedStateRegistry to CoordinationState constructor

Reply:
Yes we do that. I will remove both persistedState and remotePersistedState arguments.